services/web/server/src/simcore_service_webserver/socketio/handlers_utils.py

```diff
@@ -1,13 +1,35 @@
 import inspect
 from functools import wraps
 from types import ModuleType
-from typing import Callable
+from typing import Any, Awaitable, Callable, Union
 
 from aiohttp import web
 
 from .server import APP_CLIENT_SOCKET_DECORATED_HANDLERS_KEY, get_socket_server
 
-socketio_handlers_registry = []
+# The socket ID that was assigned to the client
+SocketID = str
+# The environ argument is a dictionary in standard WSGI format containing the request information, including HTTP headers
+EnvironDict = dict[str, Any]
+# Connect event
+SocketioConnectEventHandler = Callable[
+    [SocketID, EnvironDict, web.Application], Awaitable[None]
+]
+
+# Disconnect event
+SocketioDisconnectEventHandler = Callable[[SocketID, web.Application], Awaitable[None]]
+
+# Event
+AnyData = Any
+SocketioEventHandler = Callable[[SocketID, AnyData, web.Application], Awaitable[None]]
+
+_socketio_handlers_registry: list[
```

Comment:
Q: should we not remove that from the global scope?

Reply:
yes, I agree. I prefer using app scope perhaps but in a different PR (this is some cleanup i did for https://github.com/ITISFoundation/osparc-simcore/issues/3387)